q2_quality_control/plugin_setup.py

```diff
@@ -301,3 +302,112 @@
     description='Build bowtie2 index from reference sequences.',
     citations=[citations['langmead2012fast']]
 )
+
+# Decontam Actions
+plugin.methods.register_function(
+    function=decontam_identify,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={'metadata': Metadata,
+                'method': Str % Choices(_DECON_METHOD_OPT),
+                'freq_concentration_column': Str,
+                'prev_control_column': Str,
+                'prev_control_indicator': Str},
+    outputs=[('decontam_scores', FeatureData[DecontamScore])],
+    input_descriptions={
+        'table': ('ASV or OTU table which contaminate sequences '
+                  'will be identified from')
+    },
+    parameter_descriptions={
+        'metadata': ('metadata file indicating which samples in the '
+                     'experiment are control samples, '
+                     'assumes sample names in file correspond '
+                     'to the `table` input parameter'),
+        'method': ('Select how to which method '
+                   'to id contaminants with;\n'
```

Comment:
These newline characters confuse `q2cli`:

![image](https://user-images.githubusercontent.com/3976804/227377795-5b8c67c9-0ca9-4ea6-adc1-72df83c3f52d.png)

I would go with a semi-colon list without the newlines.

But this isn't a huge deal to be honest.